example/Swift/PrebidDemo/PrebidDemoTests/PBViewTool.swift

```diff
@@ -87,17 +87,21 @@ class PBViewTool: NSObject {
 
         for view in subviews {
             let name: String = String(describing: type(of: view))
-            if (name == "GADNWebAdView" || name == "GADOAdView") {
+            if (name == "GADNWebAdView" || name == "GADOAdView" || name == "GADWebAdView") {
                 let views  = view.subviews
                 for innerView in views {
                     let nameInner: String = String(describing: type(of: innerView))
                     if (innerView.isKind(of: WKWebView.self)) {
-                      return PBViewTool.checkJSExistInWebView(webView: innerView as! WKWebView)
+                      return PBViewTool.checkJSExistInWebView(wkWebView: innerView as! WKWebView)
+                    } else if (innerView.isKind(of: UIWebView.self)) {
```

Comment:
We are supporting iOS 10 & above... Why are we checking for UIWebView?

Reply:
It seems that AdManager uses UIWebView

<img width="1440" alt="Screen Shot 2019-04-18 at 7 07 46 PM" src="https://user-images.githubusercontent.com/8141931/56375219-99721c00-620d-11e9-8364-ecffb94c51ff.png">

iPhone 7 
iOS 12.2
Google-Mobile-Ads-SDK v7.34.0

Reply:
sounds good